### PR TITLE
QueryResource: Fix incorrect request log start times.

### LINKED ISSUE #4655: Wrong time report in DruidRequest log
https://github.com/druid-io/druid/pull/3954/files#diff-0df117b4522295642fb19e7684e38a67R354  introduced a change in the timing mechanism of druid request results fixed in https://github.com/druid-io/druid/pull/4561#issuecomment-320686484 . The result is that the reported time is nanoseconds since `<<arbitrary time point>>` applied as nanoseconds since epoch.

For most systems this makes a query report that it was issued sometime in the 1970's.

### PR DESCRIPTION
**Pull request by gianm**
A similar fix is included in #4561 but this patch has been cut down
for the 0.10.1 branch.

Fixes #4655.

### CODE REVIEW

**Comment by drcrallen** [comment]
@gianm can you just take the `startMs` directly? 

Or does `TimeUnit.MILLISECONDS.toMillis(startMs)` do something special?

File: server/src/main/java/io/druid/server/QueryResource.java [hunk 1]
```diff
@@ -185,6 +185,7 @@ public Response doPost(
       @Context final HttpServletRequest req // used to get request content-type, remote address and AuthorizationInfo
   ) throws IOException
   {
+    final long startMs = System.currentTimeMillis();
```
Comment (leventov) [inline server/src/main/java/io/druid/server/QueryResource.java]: Also I think better to call the variable `startTimestamp` to make it harder to mess it with `startNs`, but probably doesn't matter for the backport branch.

Comment (gianm) [inline server/src/main/java/io/druid/server/QueryResource.java]: Renamed it.

**Comment by gianm** [comment]
> Or does TimeUnit.MILLISECONDS.toMillis(startMs) do something special?

@drcrallen It doesn't do anything special, I removed it.

**Pull request merged**